src/api/routes/auth.py

```
@@ -0,0 +1,306 @@
+from flask import Blueprint, request, jsonify, current_app
+from flask_jwt_extended import jwt_required, get_jwt_identity
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
+from marshmallow import Schema, fields, validate
+from ..services.auth_service import AuthService
+from ..services.email_service import EmailService
+from ..models.user import User, AuditLog, db
+import jwt
```

Ruff (0.11.9)

`jwt` imported but unused

Remove unused import: `jwt`

(F401)